colors/ayu.vim

```diff
@@ -323,6 +323,25 @@ call ayu#hi('xmlEqual', 'operator', '')
 
 " }}}
 
+" HTML:" {{{
+
+" open - close tag
+call ayu#hi('htmlTag', 'tag', '')
+call ayu#hi('htmlTagName', 'entity', '')
+call ayu#hi('htmlTagN', 'entity', '')
+call ayu#hi('htmlEndTag', 'tag', '')
+
+" attribute
+call ayu#hi('htmlArg', 'operator', '')
```

Comment:
Change `operator` to `accent` to be even more consistent with VSCode & Sublime Text and we're good after that!